pkgs/servers/invidious/default.nix

```diff
@@ -102,15 +91,16 @@ crystal.buildCrystalPackage rec {
   # environment variable. Even though the database and hmac_key are
   # bogus, --help still works.
   installCheckPhase = ''
-    INVIDIOUS_CONFIG="$(cat <<EOF
+    export INVIDIOUS_CONFIG="$(cat <<EOF
     database_url: sqlite3:///dev/null
     hmac_key: "this-is-required"
     EOF
-    )" $out/bin/invidious --help
+    )"
+    $out/bin/invidious --help
+    $out/bin/invidious --version
```

Comment:
no need to chage, but this is usually done via https://nixos.org/manual/nixpkgs/stable/#tester-testVersion

Reply:
Hmm not really. This outputs a JSON containing, among other pieces of data, a version like `2024.02.18-e8a3698`, which doesn't match the package version of `0.20.1-unstable-2024-02-18`. This was more of a debug output for myself to make sure the changes to how the version is parsed and substituted in the source code are correct.

Reply:
Fair enough,  but you could make it try to grep for a date but it's probably not worth it